packages/widget-chat/src/containers/notifications/index.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import {Provider} from 'react-redux';
+
+import store from '../../store';
+
+import Notifications from '.';
+
+describe(`Notifications container`, () => {
+  it(`renders properly`, () => {
```

Comment:
No exactly sure how to unit test these renderless components. Lots of changes in store/state would need to happen.